tensorboard/webapp/metrics/views/card_renderer/scalar_card_data_table.ts

```diff
@@ -197,6 +209,51 @@ export class ScalarCardDataTable {
               selectedStepData.PERCENTAGE_CHANGE =
                 (closestEndPoint.y - closestStartPoint.y) / closestStartPoint.y;
               continue;
+            case ColumnHeaderType.STEP_AT_MAX:
+              if (!closestEndPointIndex) {
+                continue;
+              }
+              selectedStepData.STEP_AT_MAX = this.getMaxValueInRange(
+                datum.points,
+                closestStartPointIndex,
+                closestEndPointIndex,
+                true
```

Comment:
Can you add the argument name as a comment before boolean arguments. In this case "/* smoothed= */". And same with other calls to getMaxValueInRange() and getMinValueInRange().

It's worth noting, though, that no calls to getMaxValueInRange or getMinValueForRange actually pass false. So is the `smoothed` argument and `maybeSmoothedValue` even necessary?